mangaScrape.py

```diff
@@ -40,7 +31,8 @@ def saveImg(data):
     if len(check_link.split('.')) > 2:
         check_link = check_link.split('.')[1] + "." + check_link.split('.')[2]
     for link in imglinks:
-        if check_link in str(link['src']) and sm(None, str(link['src']), BASE_URL.replace("http", "")).ratio() > maxval:
+        if check_link in str(link['src']) and sm(None, str(link['src']), \
```

Comment:
It is better to refactor the second expression in separate method which returns a boolean. https://sourcemaking.com/refactoring/simplifying-conditional-expressions